applications/ContactStructuralMechanicsApplication/CMakeLists.txt

```diff
@@ -63,6 +63,7 @@ endif(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
 if(USE_COTIRE MATCHES ON)
     ## The interface_preprocess is not compatible with cotire (solve problem with explicit instantations)
     set_source_files_properties (${CMAKE_CURRENT_SOURCE_DIR}/custom_utilities/interface_preprocess.cpp PROPERTIES COTIRE_EXCLUDED TRUE)
+    set_source_files_properties (${CMAKE_CURRENT_SOURCE_DIR}/custom_conditions/*.cpp PROPERTIES COTIRE_EXCLUDED TRUE)
```

Comment:
This doesn't work I think
=> guess you need to loop the files and set the properties explicitly

Reply:
sth like 
~~~
file(GLOB condition_files ${CMAKE_CURRENT_SOURCE_DIR}/custom_conditions/*.cpp)

foreach(condition_file ${condition_files})
set_source_files_properties (condition_file PROPERTIES COTIRE_EXCLUDED TRUE)
endforeach(condition_file)
~~~